lib/Quantization/Base/Base.cpp

```diff
@@ -1,5 +1,6 @@
 /**
  * Copyright (c) 2017-present, Facebook, Inc.
+ * Copyright (c) 2019-present, NXP Semiconductor, Inc.
```

Comment:
this should not be here based on the CLA agreement.

Reply:
I understand. I removed it.